noir-projects/noir-protocol-circuits/crates/types/src/address/aztec_address.nr

```diff
@@ -110,14 +163,57 @@ impl AztecAddress {
 }
 
 #[test]
-fn compute_address_from_partial_and_pub_keys_hash() {
+fn compute_address_from_partial_and_pub_keys() {
+    let public_keys = PublicKeys {
+        npk_m: NpkM {
+            inner: Point {
+                x: 0x22f7fcddfa3ce3e8f0cc8e82d7b94cdd740afa3e77f8e4a63ea78a239432dcab,
+                y: 0x0471657de2b6216ade6c506d28fbc22ba8b8ed95c871ad9f3e3984e90d9723a7,
+                is_infinite: false,
+            },
+        },
+        ivpk_m: IvpkM {
+            inner: Point {
+                x: 0x111223493147f6785514b1c195bb37a2589f22a6596d30bb2bb145fdc9ca8f1e,
+                y: 0x273bbffd678edce8fe30e0deafc4f66d58357c06fd4a820285294b9746c3be95,
+                is_infinite: false,
+            },
+        },
+        ovpk_m: OvpkM {
+            inner: Point {
+                x: 0x09115c96e962322ffed6522f57194627136b8d03ac7469109707f5e44190c484,
+                y: 0x0c49773308a13d740a7f0d4f0e6163b02c5a408b6f965856b6a491002d073d5b,
+                is_infinite: false,
+            },
+        },
+        tpk_m: TpkM {
+            inner: Point {
+                x: 0x00d3d81beb009873eb7116327cf47c612d5758ef083d4fda78e9b63980b2a762,
+                y: 0x2f567d22d2b02fe1f4ad42db9d58a36afd1983e7e2909d1cab61cafedad6193a,
+                is_infinite: false,
+            },
+        },
+    };
+
+    let partial_address = PartialAddress::from_field(
+        0x0a7c585381b10f4666044266a02405bf6e01fa564c8517d4ad5823493abd31de,
+    );
+
+    let address = AztecAddress::compute(public_keys, partial_address);
+    let expected_computed_address_from_partial_and_pubkey =
+        0x24e4646f58b9fbe7d38e317db8d5636c423fbbdfbe119fc190fe9c64747e0c62;
+    assert(address.to_field() == expected_computed_address_from_partial_and_pubkey);
```

Comment:
There should either be a comment or at least conveyed through the name that this value is autogenerated by a TS script.